Dockerfile

```diff
@@ -1,5 +1,34 @@
+ARG VERSION=unspecified
+
 FROM python:3.6-stretch
```

Comment:
I wasn't able to build this Docker image with `python:3.6-buster`.  I believe this is simply because some Debian packages have changed names, but since it wasn't easy greasy I decided to leave it for another day.